renn/data/datasets.py

```diff
@@ -93,6 +101,69 @@ def load_csv(name, split, preprocess_fun, filter_fn=None, data_dir=None):
   return dset
 
 
+def paracrawl(language_pair,
+              vocab_files,
+              sequence_length,
+              batch_size=64,
+              transform_fn=utils.identity,
+              filter_fn=None,
+              data_dir=None):
+  """Loads any of the paracrawl datasets of parallel corpora
+  for translation from TensorFlow Datasets
+  (see https://www.tensorflow.org/datasets/catalog/para_crawl)
+
+  Arguments:
+    language_pair - str, e.g. 'ende', specifying both languages
```

Comment:
nit: We use [numpy docstring style](https://numpydoc.readthedocs.io/en/latest/format.html), which uses colons instead of hyphens when listing arguments

e.g.:

```
def foo(x, y):
  """Foo function.

  Arguments:
    x: int, first argument.
    y: int, second argument.
  """
```